Week-1/aryan29/api.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import bs4
+import sys
+from termcolor import colored,cprint
+import json
```

Comment:
You should think about correcting the import order.

https://www.python.org/dev/peps/pep-0008/#imports

Also, why is `colored` imported? Are you using it anywhere ?